### ISSUE TITLE
`npx --version` fails

### ISSUE THREAD

**Issue by konomae** [moonrepo/proto#89]
## Precondition

```
proto --version
proto 0.7.2

proto list node
18.16.0
```

## Actual

```
npx --version
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/--version - Not found
npm ERR! 404
npm ERR! 404  '--version@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user/.npm/_logs/2023-04-27T05_08_59_789Z-debug-0.log
```

## Expected

```
npx --version
9.6.4
```

**Comment by milesj** [comment]
Figured it out here: https://github.com/moonrepo/proto/pull/90

**Comment by konomae** [comment]
Thank you for investigating!

**Comment by milesj** [comment]
Fixed in 0.8

**Comment by konomae** [comment]
Thank you!
